components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```diff
@@ -9976,24 +9961,6 @@ components:
           additionalProperties:
             type: string
           description: Map of custom properties related to API revenue
-    MediationPolicy:
```

Comment:
We should not remove this. This will be a breaking change to the migration customers.

Reply:
Fixed.